src/components/calculator.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="c-wrapper">
+          <div className="ctc c-screen">
+            <div className="c-history-answer">
+              <i className="fa-solid fa-clock" />
+            </div>
+            <div className="c-answer" />
+          </div>
+
+          <div className="ctc c-compute">
+            <div className="number">
+              <p />
+            </div>
+          </div>
+
+          <div className="c-grid">
+            <button
+              type="button"
+              id="AC"
+              className="top-btn"
+              value="ac"
+              onClick={null}
+            >
+              ac
+            </button>
+            <button
+              type="button"
+              id="+/-"
+              className="top-btn"
+              value="+/-"
+              onClick={null}
+            >
+              +/-
+            </button>
+            <button
+              type="button"
+              id="%"
+              className="top-btn"
+              value="%"
+              onClick={null}
+            >
+              %
+            </button>
+            <button
+              type="button"
+              id="÷"
+              className="top-btn special"
+              value="÷"
```

Comment:
- [ ] I think it would be better if you improve your design a little bit in the way it matches the given template design. e.g.: adding lines separating different buttons,... Right now you can;t know where stats or ends a button.
![Screenshot from 2023-02-16 06-31-00](https://user-images.githubusercontent.com/83042742/219269302-e7410fe9-ac2d-4eb8-890b-64d0475cabe5.png)

Reply:
added css border to rule the lines